### PR TITLE
5 create resources sub pages

### PR DESCRIPTION
**Pull request by angelinetu**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Created two buttons in seekhelp for california and national resources that lead to a new page which has dummy filter buttons on the left and then resources pulled from supabase on the left.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @philipye314

### CODE REVIEW

File: src/screens/SeekHelp/ResourceList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Text, TouchableOpacity, View } from 'react-native';
```
Comment (philipye314) [inline src/screens/SeekHelp/ResourceList/index.tsx]: get rid of the button

File: src/screens/SeekHelp/ResourceList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            <Text>Loading...</Text>
+          )}
+        </View>
+      </View>
+    </View>
+  );
+}
+// return(
+//   <View style={styles.container}>
+
```
Comment (philipye314) [inline src/screens/SeekHelp/ResourceList/index.tsx]: can you delete these comments lol

File: src/screens/SeekHelp/index.tsx [hunk 3]
```diff
@@ -1,9 +1,40 @@
-import React, { useEffect, useState } from 'react';
-import { Button, Text, View } from 'react-native';
-import { getSeekHelpData } from '@/supabase/queries/generalQueries';
-import { Resource } from '@/types/types';
+// import React, { useEffect, useState } from 'react';
+// import { Button, Text, View } from 'react-native';
+// import styles from './styles';
```
Comment (philipye314) [inline src/screens/SeekHelp/index.tsx]: delete your comments 🥇

**Review by philipye314 — DISMISSED** [review]
LOOKS GOOD!

**Review by philipye314 — APPROVED** [review]
GOOD JOB PHILIP!
